src/renderer/coremods/experiments/plaintextPatches.ts

```diff
@@ -1,13 +1,24 @@
+import { settings } from "@replugged";
```

Comment:
This is the cause of the crashes

Reply:
Okay then. Is there no alias for this crap?

Reply:
I think a new api or a rework to the settings api as to be done. atm the general page doesn't load at all because I think settings.init wants an existing plugin id (in fact I tried to replace rp-settings with a plugin id and the page loads). For the experiments crash I have no idea tbh lol